### ISSUE TITLE
how do i disable the gui?

### ISSUE THREAD

**Issue by Dann50** [ivangeevo/crippled#2]
how do i disable the "hungry", "crippled", "hurt" etc

**Comment by ivangeevo** [comment]
You can't do it right now, but I'll consider adding it as a feature in the next updates

**Comment by Dann50** [comment]
> You can't do it right now, but I'll consider adding it as a feature in the next updates

Is it possible for you to make this mod server-side? (without the client side things, just the hunger)

**Comment by ivangeevo** [comment]
That won't work for what the mod tries to do, since the status text is rendered on the client side.

**Comment by Dann50** [comment]
> That won't work for what the mod tries to do, since the status text is rendered on the client side.

Can you make an alternative mod for server where there's no client side text? sorry if i wasn't being clear earlier

**Comment by ivangeevo** [comment]
No, I'm not making a separate mod just for that. The latest version of the mod has options to disable the hunger/health penalties if that works for you.
